tests/Fixtures/TestBundle/Entity/SoMany.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiFilter;
+use ApiPlatform\Core\Annotation\ApiResource;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\OrderFilter;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\RangeFilter;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ApiResource(attributes={
+ *     "pagination_partial"=true,
+ *     "pagination_via_cursor"={
+ *         "field"="id",
+ *         "direction"="DESC"
+ *     }
+ * })
+ *
+ * @ApiFilter(RangeFilter::class, properties={"id"})
```

Comment:
so these filters *must* match the `pagination_via_cursor`. should there be a check for that?

Reply:
Actually, shouldn't we put these filters automatically based on the `pagination_via_cursor` configuration?

Reply:
Yes. Think so!

Reply:
I've looked at it, will do that in another PR while refactoring the pagination actually.